config/webpack.common.js

```diff
@@ -135,6 +138,45 @@ module.exports = {
 			files: '**/*.css',
 		} ),
 
+		new SVGSpritemapPlugin( [
+			settings.SVGSpritemapConfig.input
+		], {
+			output: {
+				filename: 'svg/svg-sprite.svg',
+				svg4everybody: false,
+				svgo: {
+					plugins: [
+						{
+							cleanupIDs: {
+								remove: false,
+								minify: false,
+							}
+						},
+						{
+							prefixIds: {
+								delim: '-',
+								prefix: false
+							}
+						},
+						{
+							addAttributesToSVGElement: {
+								attribute: 'aria-hidden="true"'
+							}
+						},
+						{ removeRasterImages: true },
+						{
+							removeAttrs: {
+								attrs: '*:(stroke|fill):((?!^none$).)*'
+							}
+						}
+					]
+				}
+			},
+			sprite: {
+				prefix: false
```

Comment:
@colorful-tones we need a prefix in sprite file because it generates id values that have to be unique, especially if that file gets loaded into the DOM